docs/hub/dduf.md

```diff
@@ -0,0 +1,152 @@
+
+For more flexibility, you can use [`export_entries_as_dduf`] and pass a list of files to include in the final DDUF file:
+
+```python
+# Export specific files from the local disk.
+>>> from huggingface_hub import export_entries_as_dduf
+>>> export_entries_as_dduf(
+...     dduf_path="stable-diffusion-v1-4-FP16.dduf",
+...     entries=[ # List entries to add to the DDUF file (here, only FP16 weights)
+...         ("model_index.json", "path/to/model_index.json"),
+...         ("vae/config.json", "path/to/vae/config.json"),
+...         ("vae/diffusion_pytorch_model.fp16.safetensors", "path/to/vae/diffusion_pytorch_model.fp16.safetensors"),
+...         ("text_encoder/config.json", "path/to/text_encoder/config.json"),
+...         ("text_encoder/model.fp16.safetensors", "path/to/text_encoder/model.fp16.safetensors"),
+...         # ... add more entries here
+...     ]
+... )
+```
+
+This works well if you've already saved your model on the disk. But what if you have a model loaded in memory and want to serialize it directly into a DDUF file? `export_entries_as_dduf` lets you do that by providing a Python `generator` that tells how to serialize the data iteratively:
+
+```python
+(...)
+
+# Export state_dicts one by one from a loaded pipeline
+>>> def as_entries(pipe: DiffusionPipeline) -> Generator[Tuple[str, bytes], None, None]:
+...     # Build a generator that yields the entries to add to the DDUF file.
+...     # The first element of the tuple is the filename in the DDUF archive. The second element is the content of the file.
+...     # Entries will be evaluated lazily when the DDUF file is created (only 1 entry is loaded in memory at a time)
+...     yield "vae/config.json", pipe.vae.to_json_string().encode()
+...     yield "vae/diffusion_pytorch_model.safetensors", safetensors.torch.save(pipe.vae.state_dict())
+...     yield "text_encoder/config.json", pipe.text_encoder.config.to_json_string().encode()
+...     yield "text_encoder/model.safetensors", safetensors.torch.save(pipe.text_encoder.state_dict())
+...     # ... add more entries here
+
+>>> export_entries_as_dduf(dduf_path="my-cool-diffusion-model.dduf", entries=as_entries(pipe))
+```
+
+## F.A.Q.
+
+### Why build on top of ZIP?
+
+ZIP provides several advantages:
+- Universally supported file format
+- No additional dependencies for reading
+- Built-in file indexing
+- Wide language support
+
+Why not use a TAR with a table of contents at the beginning of the archive? See explanation [in this comment](https://github.com/huggingface/huggingface_hub/pull/2692#issuecomment-2519863726).
+
+### Why no compression?
+
+- Enables direct memory mapping of large files.
+- Ensures consistent and predictable remote file access.
+- Prevents CPU overhead during file reading.
+- Maintains compatibility with safetensors.
+
+### Can I modify a DDUF file?
+
+No. For now, DDUF files are designed to be immutable. To update a model, create a new DDUF file.
```

Comment:
```suggestion
No. For now, DDUF files are designed to be immutable. To update a model, create a new DDUF file.

### Which all frameworks/ apps support DDUFs?

1. [diffusers](https://github.com/huggingface/diffusers)

We are continuously reaching out to other libraries and frameworks, if you are interested in adding support for your project, open an Issue in [huggingface_hub](https://github.com/huggingface/huggingface_hub/issues) library.
```

I think adding a section like this would be helpful.